en/flight_controller/kakutef7.md

```diff
@@ -97,12 +96,13 @@ The firmware can be installed in any of the normal ways:
 
 ## Configuration
 
+If you use a 4-in-1 ESC with Betaflight/Cleanflight motor assignment you can use the [Actuator](../config/actuators.md) configuration UI to set the motor ordering appropriately.
```

Comment:
Thanks @bkueng . I don't understand what the Betaflight assignment means but I assume it is some kind of standard output wiring to motor assignment, and that this doesn't matter to us since we can set any function to any output in software.

If so, there is nothing much other than this to say, so I am merging.
Thanks!

Reply:
It's the motor ordering: https://learnassets.getfpv.com/learn/wp-content/uploads/2019/06/26225548/Motor-Order.jpg. But yes, it doesn't matter, it's just different from the PX4 default.